### PR TITLE
Mount the drives on request

### PR DESCRIPTION
**Pull request by Praveenrajmani**
- Listen for RequestedFormat changes in DirectCSIDrive object updates
  and mount the drives accordingly
- Formatting will be done separately. These changes will assume that the drives are already formatted before presenting.

Verified the functionality by updating the yaml and applying it directly (declarative). And checking if the drive status is changing to "online" after applying.

### CODE REVIEW

**Review by wlan0 — APPROVED** [review]

**Comment by wlan0** [comment]
I'll wait for @nitisht's approval before merging

**Review by nitisht — APPROVED** [review]

**Pull request merged**
